app/questionnaire/questionnaire_schema.py

```diff
@@ -888,3 +888,34 @@ def _get_rules_section_dependencies(
                 )
 
         return rules_section_dependencies
+
+    def get_summary_item_for_list_for_section(
+        self, *, section_id: str, list_name: str
+    ) -> Optional[ImmutableDict]:
+        if summary := self.get_summary_for_section(section_id):
+            for item in summary.get("items", {}):
+                if item.get("for_list") == list_name:
+                    return item  # type: ignore
+
+    def get_related_answers_for_list_for_section(
+        self, *, section_id: str, list_name: str
+    ) -> Optional[tuple[ImmutableDict]]:
+        if item := self.get_summary_item_for_list_for_section(
+            section_id=section_id, list_name=list_name
+        ):
+            return item.get("related_answers")
+
+    def get_item_label(self, section_id: str, list_name: str) -> Optional[str]:
+        if summary := self.get_summary_for_section(section_id):
+            for item in summary.get("items", []):
+                if item["for_list"] == list_name and item.get("item_label"):
+
+                    return str(item["item_label"])
+
+    def get_item_anchor_answer_id(
+        self, section_id: str, list_name: str
+    ) -> Optional[str]:
+        if summary := self.get_summary_for_section(section_id):
+            for item in summary.get("items", []):
+                if item["for_list"] == list_name and item.get("item_anchor_answer_id"):
+                    return f"#{str(item['item_anchor_answer_id'])}"
```

Comment:
Either rename to `get_item_anchor` or actually return an `ID` and create the anchor where you need it. I'll leave it up to you.

Reply:
Renamed this 👍